src/Components/Projects/Projects.tsx

```diff
@@ -110,7 +110,7 @@ const handleClose = () => setModalState("close");
 
       </div>
           </td>
-          <td className='datepicker'>{pro?.creationDate}</td>
+          <td className='datepicker' >{pro?.creationDate}</td>
 <td> <div className="dropdown ">
   <button className="btn btn-transparent dropdown-toggle " type="button" data-bs-toggle="dropdown" aria-expanded="false">
```

Comment:
The Dropfown  menu items should also be buttons instead of icons. We can make an icon clickable by adding onClick event to it, but this is not a good practice. and we are losing a lot of features by not using a button for that like using tap to navigate between interactive elements, and pressing space to click it, and more.

You should instead make a button and nest the icon inside it, and the button should be giving the onClick event.